### PR TITLE
logs: Add redirection to services page

### PR DESCRIPTION
**Pull request by marusak**
Both commits need tests but I would like to hear options about this first.

motdredirect

### CODE REVIEW

**Comment by KKoukiou** [comment]
I am happy with this addition, I myself often have this services page -> logs -> services page path.

**Comment by marusak** [comment]
I think I should filter out units we don't show. That involves:
1. `.scope`, `.mount`... This is easy, I'll just keep those 5 types we have
2. Units that are not present for any reason (maybe because it does not exists anymore). This is tricky. We would need to always load all units to check. I think we can just show warning that the unit was not found.

**Comment by marusak** [comment]
Adding release-note label but not the most important or interesting thing. Mostly if we have not much for release notes this can be used.

**Comment by marusak** [comment]
TestTimer.testCreate is impossible to get green on firefox. Since here it is affected test, it is 3 times as hard.

File: test/verify/check-system-services [hunk 1]
```diff
@@ -263,6 +263,11 @@ Unit=test.service
         b.go("/system/services#test.service")
         b.enter_page("/system/services")
 
+        b.click(".cockpit-logline:contains('WORKING')")
+        b.enter_page("/system/logs")
+        b.click("button:contains('Go to test.service')")
+        b.enter_page("/system/services")
```
Comment (martinpitt) [inline test/verify/check-system-services:269]: Could this check that it actually shows the test.service details, not the overview?

**Review by martinpitt — COMMENTED** [review]
Wow, a two-digit issue! nice job :grin:

File: test/verify/check-system-services (hunk 1 above)
Comment (martinpitt) [reply test/verify/check-system-services:269]: ... asserting `window.location.hash` should work?

Comment (marusak) [reply test/verify/check-system-services:270]: Good call, added

**Review by martinpitt — APPROVED** [review]
Thanks! I tested this, works great.

**Comment by marusak** [comment]
The new test is not working. Seems weird as it is exactly how we check it a few lines above. Also it is on the right page. I'll check it out locally.

**Comment by marusak** [comment]
Fixed. Not sure why it does not work the other way around - as it should wait as well.

**Review by martinpitt — APPROVED** [review]
Ah, makes sense! `enter_page()` has a side effect, it switches the "current frame" for the subsequent tests. So this couldn't work with the previous one.

**Pull request merged**